jobs/mysql/templates/drain.sh

```diff
@@ -26,7 +26,7 @@ for NODE in "${CLUSTER_NODES[@]}"; do
   set +e
   status_code=$(curl -s -o "/dev/null" -w "%{http_code}" "$NODE:$GALERA_HEALTHCHECK_PORT")
   set -e
-  if [[ $status_code -eq 000 || $status_code -eq 200 ]]; then
+  if [ $status_code -eq 200 ]; then
```

Comment:
```suggestion
  if [[ $status_code -eq 200 ]]; then
```
Double square brackets are to be kept, in order to ensure that we can safely skip quoting around the `$status_code` variable.